client/src/components/settings/theme.tsx

```diff
@@ -4,13 +4,28 @@ import type { updateMyTheme } from '../../redux/settings/actions';
 
 import ToggleButtonSetting from './toggle-button-setting';
 
-export enum Themes {
-  Night = 'night',
-  Default = 'default'
+type themeValue = {
+  'paypalButton': string,
+  'editorTheme': string,
+  'walletButton': 'light' | 'dark'
 }
+type Themes = {
+'night': themeValue,
+'default': themeValue
+}
+export type ThemesKind = keyof Themes;
+
+export const themesMap = new Map<ThemesKind, themeValue>([
+  [
+'default', { "paypalButton": "gold", "editorTheme": "vs-custom", 'walletButton': 'light'},
+],
+[
+  'night', { "paypalButton": "white", "editorTheme": "vs-dark-custom", 'walletButton': 'dark'},
+],
+]);
```

Comment:
The main point of the previous request was to allow for a simple way to add new theme, for example if someone wanted to add `dark-green` theme.

all they need to do is add `dark-green` to the keys in

```ts
type Theme = {
'night': themeValue,
'default': themeValue,
'darkGreen': themeValue,
}
```

then add the className `dark-green-pallete` and other areas that should be account for too in the `themesMap`.

```ts
export const themesMap = new Map<ThemesKind, themeValue>([
  [
'default', { "paypalButton": "gold", "editorTheme": "vs-custom", 'walletButton': 'light'},
],
[
  'night', { "paypalButton": "white", "editorTheme": "vs-dark-custom", 'walletButton': 'dark'},
],
[
  'darkGreen', { "paypalButton": "gray", "editorTheme": "vs-dark-custom", 'walletButton': 'dark', 'palletName': 'dark-green-pallete'},
],
]);
```

Reply:
But as shown in the number of the files changed, although this "easier" to add, but not easier to maintain, and the true solution isn't to make things "simpler" in one part and hide the abstraction, but reduce the actual abstraction that caused this request to be made.